### ISSUE TITLE
Want to add content in the pet news category.

### ISSUE THREAD

**Issue by anjaliSilawat** [akshitagupta15june/PetMe#507]
I would like to add some informative and engaging content to the pet news category, providing valuable updates, tips, and articles related to pets and their well-being.
I want to work on this issue @akshitagupta15june please assign it to me
![image](https://github.com/akshitagupta15june/PetMe/assets/108583230/8b74da18-23e4-4dbd-bcef-6970b3c8920c)

**Comment by Mridulians** [comment]
I would like to add some informative and engaging content to the pet news category, providing valuable updates, tips, and articles related to pets and their well-being.
I want to work on this issue @akshitagupta15june please assign it to me

**Comment by SakshiRawat13** [comment]
Is your feature request related to a problem? Please describe.
I want to improve the UI of this page and make it more attractive

Describe the solution you'd like
Adding cards to include image and provide description of the pet newa

I'm a GSSoCC'23 contributor. please assign this issue to me @akshitagupta15june

**Comment by Peacesandy** [comment]
Please assign me this issue as GSSOC 23

**Comment by Nayanika-12** [comment]
Are you working on this @anjaliSilawat?

**Comment by Nayanika-12** [comment]
If you are not working on it then @akshitagupta15june can you please assign me this issue? I can give a full revamp to the Pet News page as I mentioned in issue #728.

**Comment by anjaliSilawat** [comment]
> Are you working on this @anjaliSilawat?

Yes, I am working on it. Due to some technical issues with my laptop, I am unable to push my code. I will resolve this issue and push my code tomorrow as soon as possible @Nayanika-12 @akshitagupta15june

**Comment by pjyoti121** [comment]
please assign me this issue under gssoc23

**Comment by kavyachandarkar** [comment]
please assign me this issue under gssoc23 , ma'am i want to update the data here as it must have been change over a year and also want to change pictures and shape of it.
